### PR TITLE
String manipulation 1

### PR DESCRIPTION
**Pull request by AnthonyUrbina**
Screen Shot 2022-09-02 at 10 01 20 AM

### CODE REVIEW

File: string-manipulation-1/src/capitalize.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+next, go though ALL the other letters that aren't at the first index and set them to lowercase
+push all of those letters to the storage container
+and return the word in its new form
+*/
+
+function capitalize(word) {
+  var upper = '';
+  for (var i = 0; i < word.length; i++) {
+    if (i === 0) {
+      upper = upper.concat(word[0].toUpperCase());
```
Comment (codingcodymiller) [inline string-manipulation-1/src/capitalize.js:18]: If you add the first letter before the loop and just start the loop at 1, you can avoid needing this conditional

File: string-manipulation-1/src/reverse-word.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ create a function that returns a string of a word in reverse order
+ create a storage container
+ pull each letter of the string into the storage container but start from the end of the word
+once finished, return the new string
+ */
+
+function reverseWord(word) {
+  var newString = '';
+  for (var i = 1; i <= word.length; i++) {
+    newString = newString.concat(word[word.length - i]);
```
Comment (codingcodymiller) [inline string-manipulation-1/src/reverse-word.js:12]: `concat` is a method intended for arrays, you should just use the `+` operator

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AnthonyUrbina** [comment]
Screen Shot 2022-09-02 at 6 22 50 PM
Screen Shot 2022-09-02 at 6 22 38 PM

File: string-manipulation-1/src/reverse-word.js (hunk 2 above)
Comment (AnthonyUrbina) [reply string-manipulation-1/src/reverse-word.js]: okay cool, will use += from now on !

File: string-manipulation-1/src/capitalize.js (hunk 1 above)
Comment (AnthonyUrbina) [reply string-manipulation-1/src/capitalize.js]: got it
efficiency: check

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**